js/src/forum/extenders/addBioToUserCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { extend } from 'flarum/common/extend';
+import UserCard from 'flarum/forum/components/UserCard';
+import UserBio from './../components/UserBio';
+
+import type Mithril from 'mithril';
+import type ItemList from 'flarum/common/utils/ItemList';
+import type User from 'flarum/common/models/User';
+
+export default function addBioToUserCard() {
+  extend(UserCard.prototype, 'infoItems', function (items: ItemList<Mithril.Children>) {
```

Comment:
I'd type `this` in the function itself - I believe this should work. (Does UserCard not have the attributes typed?)

```suggestion
  extend(UserCard.prototype, 'infoItems', function (items: ItemList<Mithril.Children>, this: any) {
```